aws-resources/src/test/java/io/opentelemetry/contrib/aws/resource/SimpleHttpClientTest.java

```diff
@@ -69,7 +68,7 @@ static class HttpsServerTest {
         new ServerExtension() {
           @Override
           protected void configure(ServerBuilder sb) {
-            sb.tls(TlsKeyPair.of(certificate.privateKeyFile(), certificate.certificateFile()));
+            sb.tls(certificate.certificateFile(), certificate.privateKeyFile());
```

Comment:
Check out what I did in core to fix this deprecation: https://github.com/open-telemetry/opentelemetry-java/pull/6877/files
```suggestion
            sb.tls(TlsKeyPair.of(certificate.privateKey(), certificate.certificate()));
```

Reply:
will fix when renovate re-opens, just want to roll this back which should never have been merged in the first place (#1545)